charts/prefect-worker/templates/_helpers.tpl

```diff
@@ -28,6 +28,8 @@ Create the name of the service account to use
 {{- define "worker.appUrl" -}}
 {{- if eq .Values.worker.apiConfig "cloud" }}
 {{- printf "https://app.prefect.cloud/account/%s/workspace/%s" .Values.worker.cloudApiConfig.accountId .Values.worker.cloudApiConfig.workspaceId | quote }}
+{{- else }}
+{{- .Values.worker.serverApiConfig.apiUrl | quote }}
```

Comment:
yeah that was actually the motivation of the previous app that adds the `appUrl` helper (#184) . I chose the api url as the fallback because I wasn't sure how to reliably grab a self-hosted app url